WordPress/Classes/ViewRelated/Reader/ReaderDetailWebviewViewController.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+
+protocol ReaderDetailView: class {
+    func render(_ post: ReaderPost)
+    func showError()
+}
+
+class ReaderDetailWebviewViewController: UIViewController, ReaderDetailView {
+    @IBOutlet weak var webView: WKWebView!
+
+    /// The coordinator, responsible for the logic
+    var coordinator: ReaderDetailCoordinator!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        coordinator.start()
+    }
+
+    func render(_ post: ReaderPost) {
+        webView.loadHTMLString(post.contentForDisplay(), baseURL: nil)
+    }
+
+    func showError() {
+        // Show error
```

Comment:
TODO?

Reply:
Yes, this is a `TODO`. But I guess the lint fails if we add `TODO` in the code :P

Reply:
Added!